react/src/components/highlight-board/SprkHighlightBoard.js

```diff
@@ -20,16 +20,9 @@ const SprkHighlightBoard = (props) => {
     variant,
     additionalClasses,
     idString,
-    analyticsString,
```

Comment:
Accidental removal here.

Reply:
This was on purpose. `analyticsString` is not a prop on this component.